docs/projects/cn0506/index.rst

```diff
@@ -0,0 +1,396 @@
+
+CN0506_RMII interface
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: cn0506_rmii.svg
+   :width: 800
+   :align: center
+   :alt: CN0506 RMII-based block diagram
+
+Resistor-based setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - MAC interface selection
+     - MACIF_SEL1(PHY A/PHY B)
+     - MACIF_SEL0(PHY A/PHY B)
+   * - RGMII RXC/TXC 2ns delay
+     - R12/R78
+     - R9/R75
+   * - RGMII RXC/TXC 2ns delay
+     - R11/R77
+     - R9/R75
+   * - MII
+     - R17/R78
+     - R8/R74
+   * - RMII
+     - R11/R77
+     - R8/R74
+
+MDI-related resistor-based setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 45 10 20 10
+   :header-rows: 1
+
+   * - Configuration
+     - MODE
+     - R_LO
+     - R_HI
+   * - Manual MDI
+     - MODE_1
+     - 10k
+     - Open
+   * - Manual MDIX
+     - MODE_2
+     - 10k
+     - 56k
+   * - Auto MDIX - Prefer MDIX
+     - MODE_3
+     - 56k
+     - 10k
+   * - Auto MDIX - Prefer MDI
+     - MODE_4
+     - Open
+     - 10k
+
+CPU/Memory interconnects addresses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The addresses are dependent on the architecture of the FPGA, having an offset
+added to the base address from HDL(see more at :ref:`architecture`).
+
+========================  ===========
+Instance                  Address
+========================  ===========
+axi_iic_main *            0x4160_0000
+axi_sysid_0               0x4500_0000
+axi_hdmi_clkgen *         0x7900_0000
+axi_hdmi_dma *            0x4300_0000
+axi_hdmi_core *           0x70e0_0000
+axi_spdif_tx_core *       0x75c0_0000
+axi_i2s_adi **            0x7760_0000
+axi_iic_fmc **            0x4162_0000
+========================  ===========
+
+.. admonition:: Legend
+   :class: note
+
+   -  ``*``  instantiated only for Zed and ZC706 carriers
+   -  ``**`` instantiated only for Zed carrier
+
+GPIO
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Software GPIO number is calculated as follows:
+
+-  Zynq-7000: if PS7 EMIOs are used, then offset is 54
+-  ZynqMP: if PS8 EMIOs are used, then offset is 78
+
+.. list-table::
+   :widths: 25 25 25 25
+   :header-rows: 2
+
+   * - GPIO signal
+     - Direction
+     - HDL GPIO EMIO
+     - Software GPIO
+   * -
+     - (from FPGA view)
+     -
+     - Zynq-7000/ZynqMP
+   * - link_st_a
+     - IN
+     - 35
+     - 89
+   * - link_st_b
+     - IN
+     - 34
+     - 88 
+   * - int_n_a
+     - IN
+     - 33
+     - 87
+   * - int_n_b
+     - IN
+     - 32
+     - 86
+   * - gpio_bd[31:0] *
+     - INOUT
+     - 31:0
+     - 85:54
+   * - gpio_bd[14:0] **
+     - INOUT
+     - 14:0
+     - 68:54
+   * - gpio_bd_i[20:8] ***
+     - IN
+     - 20:8
+     - 98:86
+   * - gpio_bd_o[7:0] ***
```

Comment:
`**` is the syntax of emphasis, you should escape when using as a plain triple asterisk, to avoid warnings
```suggestion
   * - gpio_bd_i[20:8] \***
     - IN
     - 20:8
     - 98:86
   * - gpio_bd_o[7:0] \***
```

Reply:
yes, missed the warning related to three * def.